lua/github-theme/group/modules/mini.lua

```diff
@@ -13,67 +13,149 @@ function M.get(spec, config, opts)
+    MiniClueNextKey             = { fg = spec.diag.info },
+    MiniClueNextKeyWithPostkeys = { link = 'DiagnosticFloatingError' },
+    MiniClueSeparator           = { fg = spec.diag.hint },
+    MiniClueTitle               = { link = 'FloatTitle' },
 
-    MiniJump                      = { link = 'SpellRare' },
+    MiniCompletionActiveParameter = { style = 'underline' },
 
-    MiniJump2dSpot = {
+    MiniCursorword        = { bg = spec.sel0 },
+    MiniCursorwordCurrent = { bg = spec.sel0 },
+
+    MiniDepsChangeAdded   = { link = 'diffAdded' },
+    MiniDepsChangeRemoved = { link = 'diffRemoved' },
+    MiniDepsHint          = { link = 'DiagnosticHint' },
+    MiniDepsInfo          = { link = 'DiagnosticInfo' },
+    MiniDepsMsgBreaking   = { link = 'DiagnosticWarn' },
+    MiniDepsPlaceholder   = { link = 'Comment' },
+    MiniDepsTitle         = { link = 'Title' },
+    MiniDepsTitleError    = { link = 'DiffDelete' },
+    MiniDepsTitleSame     = { link = 'DiffText' },
+    MiniDepsTitleUpdate   = { link = 'DiffAdd' },
+
+    MiniDiffSignAdd     = { fg = spec.git.add },
+    MiniDiffSignChange  = { fg = spec.git.changed },
+    MiniDiffSignDelete  = { fg = spec.git.removed },
+    MiniDiffOverAdd     = { link = 'DiffAdd' },
+    MiniDiffOverChange  = { fg = spec.fg1, bg = spec.bg3 },
+    MiniDiffOverContext = { link = 'DiffChange' },
+    MiniDiffOverDelete  = { link = 'DiffDelete' },
+
+    MiniFilesBorder         = { link = 'FloatBorder' },
+    MiniFilesBorderModified = { link = 'DiagnosticFloatingWarn' },
+    MiniFilesCursorLine     = { link = 'CursorLine' },
+    MiniFilesDirectory      = { link = 'Directory' },
+    MiniFilesFile           = { fg = spec.fg1 },
+    MiniFilesNormal         = { link = 'NormalFloat' },
+    MiniFilesTitle          = { link = 'FloatTitle' },
+    MiniFilesTitleFocused   = { fg = spec.fg1, style = 'bold' },
+
+    MiniHipatternsFixme = { fg = spec.bg0, bg = spec.diag.error, style = 'bold' },
+    MiniHipatternsHack  = { fg = spec.bg0, bg = spec.diag.warn, style = 'bold' },
+    MiniHipatternsNote  = { fg = spec.bg0, bg = spec.diag.info, style = 'bold' },
+    MiniHipatternsTodo  = { fg = spec.bg0, bg = spec.diag.hint, style = 'bold' },
+
+    MiniIconsAzure  = { fg = c.blue.bright },
+    MiniIconsBlue   = { fg = c.blue.base },
+    MiniIconsCyan   = { fg = c.cyan.base },
+    MiniIconsGreen  = { fg = c.green.bright },
+    MiniIconsGrey   = { fg = c.fg.default },
+    MiniIconsOrange = { fg = c.orange },
+    MiniIconsPurple = { fg = c.magenta.base },
+    MiniIconsRed    = { fg = c.red.base },
+    MiniIconsYellow = { fg = c.yellow.bright },
+
+    MiniIndentscopeSymbol = { link = 'Delimiter' },
+    MiniIndentscopePrefix = { style = 'nocombine' }, -- Make it invisible
+
+    MiniJump = { link = 'SpellRare' },
+
+    MiniJump2dDim        = { link = 'Comment' },
+    MiniJump2dSpot       = {
       fg = vim.o.background == 'light' and c.red.bright or c.magenta.base,
```

Comment:
`'background'` will not have been set by us when this file runs. Instead, you probably want:

```suggestion
      fg = spec.palette.meta.light and c.red.bright or c.magenta.base,
```

Reply:
Sure. That part was already present, but I've pushed an update you suggested.